functions/notify_slack.py

```diff
@@ -244,9 +244,13 @@ def filter_message_from_slack(message):
     elif message.get('Event Source', "") in ["db-instance", "db-security-group", "db-parameter-group", "db-snapshot", "db-cluster", "db-cluster-snapshot"]:
       if message.get('Event Message', '') in ["Finished DB Instance backup", "Backing up DB instance", "Automated snapshot created", "Creating automated snapshot", "Creating manual snapshot", "Manual snapshot created", "Deleted manual snapshot"]:
         return True
-      if re.match("Finished copy of snapshot awsbackup\:*.+", message.get('Event Message', '')):
+      if re.match("Finished copy of snapshot awsbackup\:.*", message.get('Event Message', '')):
         return True
-      if re.match("Started copy of snapshot awsbackup\:*.+", message.get('Event Message', '')):
+      if re.match("Started copy of snapshot awsbackup\:.*", message.get('Event Message', '')):
+        return True
+      if re.match("Started copy of snapshot rds\:.*", message.get('Event Message', '')):
```

Comment:
Changed up the regex on this slightly. This will match the `:` in the snapshot name and anything after as well whereas before it wouldn't require a `:` to match and would just match everything after `rds`. I think this is good from a standpoint where a manual snapshot is the only type that can be cross-account shared and those can't be named with `:` so we would get those messages which we would want to know about. Whereas if it has a `:` in the name, it's system generated and can only be copied cross region/same account or is being handled by aws-backup service and the manual snap is short-lived (deleted by aws-backup right after it performs it's cross account copy).